data/human/human missions.txt

```diff
@@ -7410,3 +7410,788 @@ mission "Adelita 5"
+
+			label words
+			`	"She's not quite got the hang of words yet. But they'll come." Ildico says approvingly.`
+			
+			label onlyyou
+			`	Ilidco puts a hand on Randalin's shoulder and draws her close. She turns her attention back to the battlefield.`
+			`	"This is only the beginning, <first>." Her eyes blaze as she says, "House Drifa will rise again. You'll see. You'll be there."`
+			`	She glances back toward her mulling forces. "Thanks again for this, it means a lot that I can count on you, but I have to work to do. Until next time."`
+			`	Ildico returns to her group, barking orders and gesturing at the rubble and leaving you to your own devices.`
+
+
+mission "News Queen"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		near "Arneb" 10
+	to offer
+		has "Ice Queen 8: done"
+		random < 3
+	on offer
+		conversation
+			`A news kiosk is doing a segment on the growing threat of pirates around Haven. A certain name catches your eye, and you read:`
+			`	Self-styled "pirate queen" Ildico Drifa continues to expand her territory and reach. While there is some concern this growing centralization of power could result in greater threats to trade and stability in the Far North, most analysts think it is quite unlikely.`
+			`	Professor Grenwhich Millstone, foremost authority on the Economy and History of the Northern Pariah worlds, had this to say about Drifa:`
+			`	"It is patently inconceivable that any single magnate could consolidate power on Haven. It is culturally anathema, and economically impractical. Frankly, all this talk of a so-called 'pirate queen' is just sensationalist journalism trying to sell the news. In a few weeks another pirate captain will be the new boogieman, and then another."`
+			`	The segment ends, and the news kiosk automatically scrolls down to the next article.`
+				decline
+
+
+mission "Skadenga: Blood of Surtr"
+	landing
+	source "Windblain"
+	to offer
+		has "Ice Queen 6: done"
+		has "outfit: Blood of Surtr"
+	on offer
+		conversation
+			`As your ship descends through Windblain's cloudy atmosphere, you consider landing near Hrithfjall and asking the Ondurdis about the Blood of Surtr.`
+			choice
+				`	(Take the Blood of Surtr to Hrithfjall.)`
+					goto hrithland
+				`	(Ask about the Blood of Surtr another time.)`
+				`	(Don't visit Hrithfjall at all.)`
+					decline
+
+			`	You fly past Hjrithfjall without landing and continue on your way.`
+				defer
+
+			label hrithland
+			`	You pass numerous new homes built of driftwood and sea-stone as you approach the old village center of Hrithfjall. There is laughter and song in the air and children running everywhere; it always seems larger every time you visit it.`
+			`	At the old village hall you find the Ondurdis and Aldfrith in conversation with a band of people who quite stick out here, foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`
```

Comment:
```suggestion
			`	At the old village hall you find the Ondurdis and Aldfrith in conversation with a band of people who quite stick out here, seemingly foreigners. The Ondurdis's old eyes light up and she gives you a grandmotherly smile as you approach.`
```

Reply:
I agree that it needs to be spruced up, but I found seemingy a bit clunky. Check my recent commit and see if that is more to your likening.